freshlybuiltimagebol/downloader.py

```diff
@@ -0,0 +1,24 @@
+import requests
+from os.path import isfile
+
+class Model_File():
+	#Download the model
+	def __init__(self,model_name):
+		self.models()
+		url=self.model[model_name]
+		output_dir="freshlybuiltimagebol/models/"+model_name+".pb"
+		if isfile(output_dir):
+			print("Model already exists")
+		else:
+			ch=int(input("Do you want to download it.\n Type 1 for yes"))
+			if ch==1:
+			response = requests.get(url)
+			if response.status_code==200:
+				with open(output_model, 'wb') as file:
+					file.write(response.content)
+
+	def models(self):
+		self.model={
+				'frozen_east_text_detection':"https://github.com/ZER-0-NE/EAST-Detector-for-text-detection-using-OpenCV/blob/master/frozen_east_text_detection.pb?raw=true"
+				}
```

Comment:
Use string operation for the addition of model name in URL rather than a dictionary data structure.

Reply:
Done